table/render_automerge_test.go

```diff
@@ -413,20 +413,20 @@ func TestTable_Render_AutoMerge(t *testing.T) {
-│   │ COLUMN 1 │ COLUMN 2 │ COLUMN 3 │ COLUMN 4 │   COLUMN 5  │   COLUMN 6  │   COLUMN 7  │   COLUMN 8  │
-├───┼──────────┼──────────┼──────────┼──────────┼─────────────┴─────────────┼─────────────┴─────────────┤
-│ 1 │ a.a.a.a  │ Pod 1A   │ NS 1A    │ C 1      │  4F8F5CB531E3D49A61CF417C │  4F8F5CB531E3D49A61CF417C │
-│   │          │          │          │          │  D133792CCFA501FD8DA53EE3 │  D133792CCFA501FD8DA53EE3 │
-│   │          │          │          │          │  68FED20E5FE0248C3A0B64F9 │  68FED20E5FE0248C3A0B64F9 │
-│   │          │          │          │          │  8A6533CEE1DA614C3A8DDEC7 │  8A6533CEE1DA614C3A8DDEC7 │
-│   │          │          │          │          │  91FF05FEE6D971D57C134832 │  91FF05FEE6D971D57C134832 │
-│   │          │          │          │          │         0F4EB42DR         │         0F4EB42DRR        │
-├───┼──────────┼──────────┼──────────┼──────────┼───────────────────────────┴───────────────────────────┤
-│ 2 │ a.a.a.a  │ Pod 1A   │ NS 1A    │ C 2      │                           Y                           │
-├───┼──────────┼──────────┼──────────┼──────────┼───────────────────────────────────────────────────────┤
-│ 3 │ a.a.a.a  │ Pod 1A   │ NS 1A    │ C 2      │                           Y                           │
-└───┴──────────┴──────────┴──────────┴──────────┴───────────────────────────────────────────────────────┘`)
+┌───┬──────────┬──────────┬──────────┬──────────┬─────────────┬──────────────┬─────────────┬──────────────┐
+│   │ COLUMN 1 │ COLUMN 2 │ COLUMN 3 │ COLUMN 4 │   COLUMN 5  │   COLUMN 6   │   COLUMN 7  │   COLUMN 8   │
+├───┼──────────┼──────────┼──────────┼──────────┼─────────────┴──────────────┼─────────────┴──────────────┤
+│ 1 │ a.a.a.a  │ Pod 1A   │ NS 1A    │ C 1      │  4F8F5CB531E3D49A61CF417C  │  4F8F5CB531E3D49A61CF417C  │
+│   │          │          │          │          │  D133792CCFA501FD8DA53EE3  │  D133792CCFA501FD8DA53EE3  │
+│   │          │          │          │          │  68FED20E5FE0248C3A0B64F9  │  68FED20E5FE0248C3A0B64F9  │
+│   │          │          │          │          │  8A6533CEE1DA614C3A8DDEC7  │  8A6533CEE1DA614C3A8DDEC7  │
+│   │          │          │          │          │  91FF05FEE6D971D57C134832  │  91FF05FEE6D971D57C134832  │
+│   │          │          │          │          │          0F4EB42DR         │         0F4EB42DRR         │
+├───┼──────────┼──────────┼──────────┼──────────┼────────────────────────────┴────────────────────────────┤
+│ 2 │ a.a.a.a  │ Pod 1A   │ NS 1A    │ C 2      │                            Y                            │
+├───┼──────────┼──────────┼──────────┼──────────┼─────────────────────────────────────────────────────────┤
+│ 3 │ a.a.a.a  │ Pod 1A   │ NS 1A    │ C 2      │                            Y                            │
+└───┴──────────┴──────────┴──────────┴──────────┴─────────────────────────────────────────────────────────┘`)
```

Comment:
It looks like there might be a tiny bug with centering, or measuring the max length. In Row 1, merged Column 5/6 and merged Column 7/8 contents can be shrunk down further by 2 spaces (1 on each side).

Reply:
Thanks! Not a deal breaker, but we may as well try while this code is fresh in our minds.